skunk/rust/skunk/src/parser/tests/expression_builder.rs

```diff
@@ -0,0 +1,70 @@
+
+use super::super::ast::{ExpressionValue, ExpressionValueEnum, Operator, OutputExpression, SafeSpan};
+
+pub struct Expr<'a> {
+  offset: usize,
+  line: u32,
+  expr: Box<dyn 'a + FnOnce(usize, u32) -> ExpressionValueEnum>,
+}
+
+impl <'a> Expr<'a> {
+  fn mk(self, offset: usize, line: u32) -> ExpressionValue {
```

Comment:
Nit: Naming suggestion `build` (as this is a builder pattern)

Reply:
The publicly visible function is called build, this is an internal helper.